### ISSUE TITLE
package.json script with `--` fails to be recognized

### ISSUE THREAD

**Issue by tkrotoff** [webpro/knip#421]
Example of package.json:
```JSON
{
  "scripts": {
    "test:seed": "pnpm --recursive --parallel test -- --sequence.seed=1700316221712"
  }
}
```

Knip 3.10.0 detects `--sequence.seed=1700316221712` as a binary while it is a [Vitest parameter](https://vitest.dev/config/#sequence-seed):
```
Unlisted binaries (1)
--sequence.seed=1700316221712  package.json
```

I guess this is because of ` -- ` before `--sequence.seed=1700316221712`

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.11.0. See [Release 3.11.0](https://github.com/webpro/knip/releases/tag/3.11.0) for release notes._

**Comment by webpro** [comment]
Thanks for the report @tkrotoff, should be good now.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v4.0.0. See [Release 4.0.0](https://github.com/webpro/knip/releases/tag/4.0.0) for release notes._
